### DISCUSSION TITLE
IgnoreFileTimestamps value isn't followed

### DISCUSSION THREAD

**Discussion by mamema** [kagemomiji/airsonic-advanced#661] [Q&A]
i've entered 

IgnoreFileTimestamps=false

in arsoinic.properties

which, thats my expectation, the files should get sorted under "new music" according the last modified timestamp, which is the (unwritten) standard also in other applications like navidrome or lyrion mediaserver. (there it's working)
But the files appear to be sorted according the "last accessed" date, which is wrong, as this value changes every day when a backup is running, so the "new music" sort is kinda "unsorted".....
It's even so, that which the kagemomiji release the IgnoreFileTimestamps value is gone completely. It is not clear to me how i get back a full scan run which sorts according the "last modified" timestamp.

Do i have to configure something different here?

**Comment by bartmans** [comment]
I can't confirm this behaviour. Which screen are you referring to? How is your music organised?

When I look at the 'most recently added' homescreen, it shows the most recently added. You can set this screen as the default.

**Comment by Faeb35** [comment]
Please have a look at the main page of this repo. 
`IgnoreFileTimestamps` got changed to `FullScan ` with Airsonic-Advaned, hence the naming was not very intuitive. 
And no, it's not doing what you're expecting. It's doing a full scan of the whole library instead of only looking for changes in your files.
Please also check out the `ClearFullScanSettingAfterScan` property
